cobbler/actions/check.py

```diff
@@ -135,7 +135,19 @@ def check_service(self, status, which, notes=""):
         if notes != "":
             notes = " (NOTE: %s)" % notes
         return_code = 0
-        if self.checked_family in ("redhat", "suse"):
+        if utils.is_systemd():
```

Comment:
The order seems to be wrong. Containers that have supervisorctl still have systemctl. Also is there a reason for separate functions? E.g. a list of binaries with iterator would be much smaller.

Reply:
That's what we have in system-tests:

```
srvctl() {
        $(command -v supervisorctl systemctl | head -1) ${@}
}
```

Reply:
I corrected the order. I introduced the functions because I need the same for service restart. So instead of having the commands in 2 places I thought it's better to have a function for it (and is_systemd was already there)